### ISSUE TITLE
Technical support for transitioning Podcast feeds into megaphone

### ISSUE THREAD

**Issue by vsinghkpcc** [SCPR/SCPRv4#1131]
AC

- [x] We have a path to importing our feeds into Megaphone

Feeds to import: https://docs.google.com/spreadsheets/d/1czawMbDfTa2Bj9LHwvu-rBPf5BQ7LURlBMNs6sooOzM/edit#gid=0

**Comment by arellaTV** [comment]
Talked with someone in Megaphone - she's asking their dev team if they can pull up the exact error. Will put this on hold for now until she gets back to me.

**Comment by arellaTV** [comment]
Dev teams wasn't able to find anything wrong with the filmweek rss feed, so they're going to take a look at it today.

In the meantime, my first instinct would then be to integrate Megaphone into our CMS (via their API).

**Comment by arellaTV** [comment]
The Frame has been imported 100% into Megaphone :tada: Turns out it was related to the way the redirect url had a different protocol (`http`) than the initial url (`https`)

There are still some stragglers in Airtalk, so they're looking into that now.

**Comment by arellaTV** [comment]
Still no notable progress on why there are certain  audio files failing. In the meantime, I accidentally found a way to import the podcast xml (when we eventually need it).

Click "Stop watching" at the top right of a podcast page. That will turn the podcast to a native podcast model, and all episodes will then be fully featured and editable. It's unclear if this is the intended behavior or not, but it could be a good idea to do the following:

Implement Megaphone tags in RSS feed -> Implement API integration into outpost -> Stop watching in Megaphone

**Comment by arellaTV** [comment]
After removing Airtalk from the podcast list and re-adding it, every file was processed successfully. I'm trying that for Take two now as well. The initial snags might have been due to their servers getting a cached response from our servers. I'll continue to monitor to see if audio processing is still an issue.

**Comment by arellaTV** [comment]
Managed to 100% our current line-up:
- Airtalk
- Filmweek
- Filmweek Marquee
- Take Two
- The Frame

Take Two had some snags, so I tried updating the Episode through the Megaphone API with
````
{
  "backgroundAudioFileUrl": "{insert the episode's url here}",
  "audioFileProcessing": true
}
````

Which seemed to have worked! Lauren from Megaphone is going to confirm tomorrow if I've done it correctly. I asked if she could let me know if her team finds out more about the audio processing issue. In the meantime, I'll keep an eye on it to make sure that there aren't any failing episodes for future episodes.

For now though, the "manual retrigger" seems like a good solution and enough for me to feel comfortable in fulfilling this ticket. I'll re-open this if it becomes a recurring issue once again.
